packages/zoe/src/zoeService/zoe.js

```diff
@@ -385,10 +386,8 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
             instanceAdmin,
             proposal,
             brandToPurse,
-            {
-              offerResult: offerResultPromiseKit.promise,
-              exitObj: exitObjPromiseKit.promise,
-            },
+            exitObjPromiseKit.promise,
```

Comment:
Didn't you already flatten this out in a previous PR? In any case, the flattening is good.

Reply:
Whoops, I've got some rebasing to do, sorry.